src/mixins/commonMixin.js

```diff
@@ -158,8 +159,30 @@ export default {
         onInput(event, type) {
             if (event.isTrusted) return;
             this.$emit(`input-${camelToKebab(type)}`, event.target.value);
+            let validForNextStep = false;
+
+            if (BRANDS_WITH_MULTIPLE_MASKS.includes(this.cardInfo.brandAlias)) {
+                const longest = this.cardInfo.allMasks.sort(function (a, b) {
+                    return b.length - a.length;
+                });
+
+                const MAX_ALLOWED_SYMBOLS = longest[0].replace(
```

Comment:
Use UPPER_CASE_SNAKE_CASE only for static data, here you are getting data dynamically

Reply:
\+